solutions/usaco-915.mdx

```diff
@@ -9,6 +9,41 @@ author: Sathvik Chundru
 
 <LanguageSection>
  
+<PySection>
+
+```py
+import sys
+
+sys.stdin = open('herding.in', 'r')
+sys.stdout = open('herding.out', 'w')
+a, b, c = map(int, input().split())
+
+# Best scenario: the three elements are already in order.
+if c == a + 2:
+	print(0)
+"""
+If there is a difference by 2,
```

Comment:
these comments are so long, maybe they would be better as a written explanation instead of in the code? what do other people think?

Reply:
it's described in the editorial, think these comments just link editorial portions to code segments

Reply:
These comments are so long... is there any way to shorten? I'll take a look later